src/meta/processors/jobMan/JobManager.cpp

```diff
@@ -233,6 +265,22 @@ void JobManager::removeExpiredJobs(const std::vector<std::string>& expiredJobsAn
     baton.wait();
 }
 
+bool JobManager::checkJobExist(const cpp2::AdminCmd& cmd,
+                               const std::vector<std::string>& paras,
+                               JobID& iJob) {
+    JobDescription jobDesc(0, cmd, paras);
+    auto it = inFlightJobs_.begin();
+    while (it != inFlightJobs_.end()) {
+        if (it->second == jobDesc) {
```

Comment:
why don't compare job type and parameter directly ?

Reply:
Here is the condition for judging that JobDescription is equal:
job type, parameter and status